packages/@aws-cdk/aws-apigateway/test/restapi.test.ts

```diff
@@ -868,163 +868,210 @@ describe('restapi', () => {
     expect(method.api).toBeDefined();
   });
 
-  describe('Import', () => {
-    test('fromRestApiId()', () => {
-      // GIVEN
-      const stack = new Stack();
+  test('RestApi minCompressionSize', () => {
+    // GIVEN
+    const app = new App({
+      context: {
+        '@aws-cdk/aws-apigateway:disableCloudWatchRole': true,
+      },
+    });
 
-      // WHEN
-      const imported = apigw.RestApi.fromRestApiId(stack, 'imported-api', 'api-rxt4498f');
+    const stack = new Stack(app);
+    const api = new apigw.RestApi(stack, 'RestApi', {
+      minCompressionSize: Size.bytes(1024),
```

Comment:
Please add a test to make sure that `minimumCompressionSize` also still works.

You might need to use `testDeprecated()`.

Reply:
Also users shouldn't be able to set both properties.

Reply:
@pattasai I don't see where the second part of this was addressed. I would expect to see an error thrown if the user specifies `minCompressionSize` and `minimumCompressionSize`, and a test to assert that that behavior happens.

Reply:
☝️ What Kaizen said. Now users are able to go:

```ts
new RestApi(..., {
  minimumCompressionSize: 666,
  minCompressionSize: Size.bytes(1024),
});
```

What do you think should happen in that case?